### PR TITLE
feat: edit WidgetDataTable style and edit widget-table-helper

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [ ] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

### CODE REVIEW

File: apps/web/src/services/dashboards/widgets/_components/WidgetDataTable.vue [hunk 1]
```diff
@@ -378,10 +378,14 @@ const handleClickRow = (rowData) => {
...
         .th-contents {
-            @apply flex justify-between pl-4;
+            @apply flex justify-between;
+            padding: 0.25rem 0 0.25rem 1rem;
             line-height: 2;
             .th-text {
                 @apply inline-flex items-center gap-1;
+                white-space: pre-line;
+                text-align: right;
+                line-height: 1.25;
```
Comment (yuda110) [inline apps/web/src/services/dashboards/widgets/_components/WidgetDataTable.vue:388]: description: for this

스크린샷 2023-06-13 오후 8 07 42

File: apps/web/src/services/dashboards/widgets/_helpers/widget-table-helper.ts [hunk 2]
```diff
@@ -37,7 +37,7 @@ export const getWidgetTableDateFields = (
     let count = 0;
     while (now.isSameOrBefore(end, timeUnit)) {
         dateFields.push({
-            name: `${fieldsKey}_sum.${count}.value`,
+            name: `${fieldsKey}.${count}.value`,
```
Comment (yuda110) [inline apps/web/src/services/dashboards/widgets/_helpers/widget-table-helper.ts:40]: description: The return value of the API used by the asset widget does not include values in such a format. (_sum)

**Review by sulmoJ — APPROVED** [review]
I hope the PR title is more clear!
LGTM!

File: apps/web/src/services/dashboards/widgets/_base/base-count-of-findings/BaseCountOfFindingsWidget.vue [hunk 3]
```diff
@@ -211,7 +211,7 @@ const drawChart = (chartData) => {
         });
         series.columns.template.onPrivate(('width'), (width, target) => {
             array.each(target?.dataItem?.bullets, (bullet) => {
-                if (width && width < 30) {
+                if ((width !== undefined) && width < 30) {
```
Comment (yuda110) [inline apps/web/src/services/dashboards/widgets/_base/base-count-of-findings/BaseCountOfFindingsWidget.vue:214]: before
스크린샷 2023-06-13 오후 10 13 08

after😎
스크린샷 2023-06-13 오후 10 13 18

**Pull request merged**